RELEASING.md

```diff
@@ -0,0 +1,17 @@
+# Releasing
+
+## Pre-release
+
+Create a pull request named `Release v<version>` that does the following:
+
+1. Update `go.mod` for submodules to depend on the new release.
```

Comment:
I want to make a submodule in `./build` like here: https://github.com/pellared/fluentassert/tree/main/build
Thanks to it we can get rid of `tools` directory and e.g. use `go-shellwords` in our build pipeline